frontend/package.json

```diff
@@ -75,7 +79,31 @@
       "react/no-array-index-key": "warn",
       "react/prefer-stateless-function": "warn",
       "react/prop-types": "warn",
-      "react/sort-comp": "warn"
+      "react/sort-comp": "warn",
+      "react/display-name": "warn",
+      "react/jsx-key": "warn",
+      "react/jsx-no-comment-textnodes": "warn",
+      "react/jsx-no-duplicate-props": "warn",
+      "react/jsx-no-target-blank": "warn",
+      "react/jsx-no-undef": "warn",
+      "react/jsx-uses-react": "warn",
+      "react/jsx-uses-vars": "warn",
+      "react/no-children-prop": "warn",
+      "react/no-danger-with-children": "warn",
+      "react/no-deprecated": "warn",
+      "react/no-direct-mutation-state": "warn",
+      "react/no-find-dom-node": "warn",
+      "react/no-is-mounted": "warn",
+      "react/no-render-return-value": "error",
+      "react/no-string-refs": "error",
+      "react/no-unescaped-entities": "warn",
+      "react/no-unknown-property": "warn",
+      "react/react-in-jsx-scope": "warn",
+      "react/require-render-return": "error",
+      "react/jsx-equals-spacing": "warn",
+      "react/jsx-tag-spacing": "warn",
+      "react/jsx-indent": "warn",
+      "react/jsx-indent-props": "warn"
```

Comment:
did you try this plugin: https://github.com/bfanger/eslint-plugin-only-warn,

found here: https://stackoverflow.com/questions/50705508/is-it-possible-to-show-warnings-instead-of-errors-on-all-of-eslint-rules

Reply:
When i read the readme it looked like it downgrades all errors to warnings. If you look at the list I left some of them as errors because stuff like `no-render-return-value` is worth breaking the compile over. I got the impression the plugin would not let us do that. But i'll try it and see if it lets us still set some as error if we want.